hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieTTLJob.java

```diff
@@ -74,7 +74,7 @@ public void run() {
     // need to do commit in SparkDeletePartitionCommitActionExecutor#execute
     try (SparkRDDWriteClient<HoodieRecordPayload> client =
              UtilHelpers.createHoodieClient(jsc, cfg.basePath, "", cfg.parallelism, Option.empty(), props)) {
-      String instantTime = client.createNewInstantTime();
+      String instantTime = client.startCommit(HoodieTimeline.REPLACE_COMMIT_ACTION);
```

Comment:
This is not right, the requested instant was created with empty `HoodieRequestedReplaceMetadata`.